package.json

```diff
@@ -30,22 +30,22 @@
   ],
   "license": "OGL-UK-3.0",
   "dependencies": {
-    "@azure/identity": "3.1.3",
-    "applicationinsights": "1.8.7",
-    "ffc-messaging": "2.6.3",
-    "joi": "17.3.0",
-    "notifications-node-client": "7.0.0",
-    "pg": "8.3.3",
-    "pg-hstore": "2.3.3",
-    "sequelize": "6.21.2"
+    "@azure/identity": "3.3.0",
+    "applicationinsights": "2.7.3",
+    "ffc-messaging": "2.7.0",
+    "joi": "17.10.0",
+    "notifications-node-client": "7.0.3",
+    "pg": "^8.11.3",
```

Comment:
Suggest not using `^` as it adds risk to behaviour for different container builds.  Also prevents premature bumps to buggy versions

Reply:
Removed ^